controllers/goharbor/internal/test/utils.go

```diff
@@ -11,6 +12,14 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+func AddVersionAnnotations(annotations map[string]string) map[string]string {
+	return version.SetVersion(annotations, GetVersion())
+}
+
+func GetVersion() string {
+	return "2.2.1"
```

Comment:
This the harbor version for the tests of core, trivy ... cr, because now the version is a required field in the spec, we need to set this info to these cr in tests.